internal/rpchelp/methods.go

```diff
@@ -50,6 +50,8 @@ var Methods = []struct {
 	{"gettransaction", []interface{}{(*btcjson.GetTransactionResult)(nil)}},
 	{"help", append(returnsString, returnsString[0])},
 	{"importprivkey", nil},
+	{"importpubkey", nil},
+	{"importaddress", nil},
```

Comment:
importaddress should come before importprivkey (here and everywhere else in the commit) so the methods remain alphabetical.